lib/twap/events/self_interval_tick.js

```diff
@@ -62,11 +62,6 @@ const onSelfIntervalTick = async (instance = {}) => {
     }
   }
 
-  if (submitDelay) {
-    const [, t] = timeout(submitDelay)
-    await t()
-  }
-
   if (instance.state.shutdown) {
```

Comment:
i think this check isn't necessary now since we aren't awaiting delay

Reply:
It has async call `exec:order:cancel:all`
https://github.com/bitfinexcom/bfx-hf-algo/blob/8d0ec3f5e6af47beeab9a0bf03fcf451a4a36108/lib/twap/events/self_interval_tick.js#L41-L47

Perhaps the `state.shutdown` could be changed while the async functions are running before this check